.github/workflows/test.yml

```diff
@@ -255,7 +255,7 @@ jobs:
       - uses: actions/upload-artifact@v4
         if: failure()
         with:
-          name: e2e
+          name: e2e_${{ matrix.job_index }}
```

Comment:
It merges, doesn't it? So we don't have to add the index.

Reply:
It doesn't merge. It fails because an artifact with the same name already exists. At least in the v4 version, I don't know how it was behaving before.

Reply:
OK, it merged before.